db/migrate/20170910091746_change_validity_period_to_float_in_triggers.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeValidityPeriodToFloatInTriggers < ActiveRecord::Migration[5.0]
+  def change
+    change_column :triggers, :validity_period, :float
```

Comment:
if you provide the former data type, this migration is reversible

Reply:
@roschaefer I couldn't find an example for that, could you provide one please?